.storybook/pages/ProjectOverview/ProjectOverview.tsx

```diff
@@ -107,7 +107,7 @@ export const ProjectOverview = ({ activeIndex = 0 }: Props) => {
               <li>How are structures in the body organized?</li>
             </ol>
           </Text>
-          <Hr className="u-margin-bottom-none" />
+          <Hr className="mb-0" />
```

Comment:
This one's also being overridden but honestly it looks way better without it, so how about we just remove the class instead?
```suggestion
          <Hr />
```